apps/src/code-studio/components/LabContainer.jsx

```diff
@@ -27,7 +27,13 @@ const LabContainer = ({children, onError}) => {
 
   return (
     <ErrorBoundary fallback={<ErrorFallbackPage />} onError={onError}>
-      <div id="lab-container" className={moduleStyles.labContainer}>
+      <div
+        id="lab-container"
+        className={classNames(
+          moduleStyles.labContainer,
+          isLabLoading && moduleStyles.labContainerLoading
```

Comment:
Not strictly related to this change, but @breville spotted a bug where sometimes clicking Start Over threw an exception because the Project Manager hadn't been set. From what I could tell, this is because the MusicView UI was still clickable while the loading screen was up, and now that we load level properties before setting the project manager, you could sometimes click through to Start Over which tried to call save on a nonexistent project manager.

There's probably more we can do within music view around disallowing certain actions if loading is in progress, but I figured regardless we shouldn't allow clicks through the loading screen.